dist/profile/manifests/openvpn.pp

```diff
@@ -51,6 +52,15 @@
     require          => [Docker::Image[$image]]
   }
 
+
+  file { '/etc/cloud/cloud.cfg.d':
+    ensure  => 'directory',
+    owner   => 'root',
+    group   => 'root',
+    recurse => true
+  }
+
+
   # Ensure cloud-init doesn't manage network (netplan config + netplan apply + systemd, in Ubuntu Bionic)
   # We assume the parent folder already exists
```

Comment:
We're testing the existence of the folder above, this comment could be removed.
```suggestion
```

Reply:
Done